plugin/webdevicons.vim

```diff
@@ -96,23 +96,24 @@ function s:getDistro()
     return s:distro
   endif
 
-  let s:lsb = system('lsb_release -i')
-
-  if s:lsb =~# 'Arch'
-    let s:distro = ''
-  elseif s:lsb =~# 'Ubuntu'
-    let s:distro = ''
-  elseif s:lsb =~# 'Cent'
-    let s:distro = ''
-  elseif s:lsb =~# 'Debian'
-    let s:distro = ''
-  elseif s:lsb =~# 'Dock'
-    let s:distro = ''
-  else
-    let s:distro = ''
+  if executable('lsb_release -i')
+    let s:lsb = system('lsb_release -i')
+    if s:lsb =~# 'Arch'
+      let s:distro = ''
+    elseif s:lsb =~# 'Ubuntu'
+      let s:distro = ''
+    elseif s:lsb =~# 'Cent'
+      let s:distro = ''
+    elseif s:lsb =~# 'Debian'
+      let s:distro = ''
+    elseif s:lsb =~# 'Dock'
+      let s:distro = ''
+    else
+      let s:distro = ''
+    endif
+    return s:distro
   endif
-
-  return s:distro
+  return
```

Comment:
This function contract should need return s:distro.

If non executable, set default value.
And at last, return s:distro.